templates/config.yaml

```
@@ -0,0 +1,23 @@
+  max_connect_pool: 30
+  min_connect_pool: 10
+  connect_timeout: 30
+  acquire_timeout: 60
+server:
+  host: "127.0.0.1"
+  port: "8080"
+indexdb_backend:
+  invite_url: "http://18.136.124.172:3100/api/event/submit"
+nostr:
+  priv_key: "nsec1ufnus6pju578ste3v90xd5m2decpuzpql2295m3sknqcjzyys9ls0qlc85"
+  ws_url: "ws://localhost:10547" 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)